### PR TITLE
Make Mac build instructions easier to find

### PR DESCRIPTION
**Pull request by intelliot**
I hope this (automatically?) adds a link to https://xrpl.org/install-rippled.html

### CODE REVIEW

File: content/infrastructure/rippled/installation/build-on-linux-mac-windows.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+parent: install-rippled.html
+blurb: Build rippled on Linux, Mac (macOS), or Windows
+labels:
+  - Core Server
+  - Blockchain
+top_nav_grouping: Popular Pages
+---
+# Build `rippled` on Linux, Mac, or Windows
+
+Building `rippled` across various platforms requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler. Familiarity with Conan and CMake will be helpful.
```
Comment (intelliot) [inline content/infrastructure/rippled/installation/build-on-linux-mac-windows.md]:
```suggestion
Building `rippled` across various platforms requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
```

File: content/infrastructure/rippled/installation/build-on-linux-mac-windows.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+labels:
+  - Core Server
+  - Blockchain
+top_nav_grouping: Popular Pages
+---
+# Build `rippled` on Linux, Mac, or Windows
+
+Building `rippled` across various platforms requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
+
+To continue, proceed to [the latest `rippled` build instructions on GitHub](https://github.com/XRPLF/rippled/blob/develop/BUILD.md).
```
Comment (mvadari) [inline content/infrastructure/rippled/installation/build-on-linux-mac-windows.md:14]: Shouldn't this still mention the packages?

Comment (intelliot) [reply content/infrastructure/rippled/installation/build-on-linux-mac-windows.md:14]: What do you mean? The packages are already documented on https://xrpl.org/install-rippled-on-centos-rhel-with-yum.html and https://xrpl.org/install-rippled-on-ubuntu.html

**Review by mDuo13 — APPROVED** [review]
Approved now that I've added the page to the build

File: content/infrastructure/rippled/installation/build-on-linux-mac-windows.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+parent: install-rippled.html
+blurb: Build rippled on Linux, Mac (macOS), or Windows
+labels:
+  - Core Server
+  - Blockchain
+top_nav_grouping: Popular Pages
+---
+# Build `rippled` on Linux, Mac, or Windows
+
+Building `rippled` across various platforms requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
```
Comment (intelliot) [inline content/infrastructure/rippled/installation/build-on-linux-mac-windows.md]:
```suggestion
Building `rippled` across various platforms such as Windows, Linux, or macOS requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
```

Take this opportunity to show that the order the platforms are listed in doesn't matter, and Mac == macOS

**Pull request merged**